### DISCUSSION TITLE
part-ii/2008-19/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#156] [Question comments]
Part II 2008 Coding and Cryptography: 2.I.4G 

https://questions.tripos.org/part-ii/2008-19/

**Comment by walrus1729** [comment]
<details>
<summary>Examples of codes</summary>

A possible Shannon-Fano code is 

$$
0.4: 00 \\
0.3: 01 \\
0.2: 100 \\
0.1: 1100 \\
$$

and a possible Huffman code is

$$
0.4: 1 \\
0.3: 00 \\
0.2: 010 \\
0.1: 011 \\
$$

</details>
